lib/factory_factory_girl/configuration.rb

```diff
@@ -16,13 +16,13 @@ def initialize
       @rails_options = {}
     end
 
-    def match(rule, value: nil, function: nil)
-      raise "Need to give attribute or process" if                 value.nil? && function.nil?
+    def match(attributes, value: nil, function: nil)
```

Comment:
I named it `rule` because that it should be a regular expression. So surely you can set an attribute name as yours rule, but in most cases it should be an naming rule like:
`/.*_name/` for all attributes that is something's name.

So I think renaming `rule` to `attributes` is not a very good idea.